mizar/networkpolicy/networkpolicy_util.py

```diff
@@ -366,23 +366,23 @@ def fill_cidrs_from_directional_traffic_rules(self, data, policy_name, indexed_p
                     self.add_namespace_label_networkpolicy(data, rule_item["namespaceSelector"]["matchLabels"], policy_name)
                     if direction == "ingress":
                         data["namespace_label_map"][indexed_policy_name] = self.get_namespace_label_values_by_label_dict(rule_item["namespaceSelector"]["matchLabels"])
-
-                    namespaces = kube_list_namespaces_by_labels(networkpolicy_opr.core_api, rule_item["namespaceSelector"]["matchLabels"])
-                    if namespaces is not None:
-                        for namespace in namespaces.items:
-                            pods = kube_list_pods_by_namespace(networkpolicy_opr.core_api, namespace.metadata.name)
-                            if pods is not None:
-                                for pod in pods.items:
-                                    self.add_pod_ip_into_cidrs_map(pod, policy_name, data["cidrs_map_no_except"][indexed_policy_name])
+                    else:
+                        namespaces = kube_list_namespaces_by_labels(networkpolicy_opr.core_api, rule_item["namespaceSelector"]["matchLabels"])
```

Comment:
same as above

Reply:
For ingress traffic, use label based policy. The label based policy only fits for ingress for now. For egress traffic, still keep original usage which is ip based policy. 
Added comments in the code to tell the difference.